modules/vmseries/variables.tf

```
@@ -30,9 +30,32 @@ variable "network_interfaces" {
 }
 
 variable "bootstrap_options" {
-  description = "VM-Series bootstrap options to pass using instance metadata."
+  description = <<-EOF
+  VM-Series bootstrap options to pass using instance metadata.
+
+  Proper syntax is a map where keys are the bootstrap properties.
```

```suggestion
  Proper syntax is a map, where keys are the bootstrap parameters.
```
I'd propose to stick to wording used in [bootstrap options doc](https://docs.paloaltonetworks.com/vm-series/10-2/vm-series-deployment/bootstrap-the-vm-series-firewall/create-the-init-cfgtxt-file/init-cfgtxt-file-components).